src/Pizzeria.cpp

```diff
@@ -49,11 +49,14 @@ void Pizzeria::bakePizzas(int orderId)
         for (const auto & pizza : pizzas)
         {
             std::cout << "Baking " << pizza->getName() << std::endl;
-            std::this_thread::sleep_for(pizza->getBakingTime());
+            //std::this_thread::sleep_for(pizza->getBakingTime());
```

Comment:
Is this comment necessary?

Reply:
I've left it for learning purpose, to checj how it looks in dummy style

Reply:
Isn't branching in Github designed for purpose of switching between versions of code? I think that's better to have two working version for learning purposes than having commented code. If you return there after long time you would need some time to figure what to comment/uncomment to get alternate version.

Reply:
done